.github/workflows/integration-tests.yml

```diff
@@ -1,9 +1,9 @@
 name: Integration tests
 on:
   push:
-    branches: [ main, master ]
-    schedule:
-      - cron: '0 0 * * *'
+    branches: [ main, master, jsign-eest-release ]
```

Comment:
Gonna remove this before merging -- this was just for testing integration tests.